doc/user/ospfd.rst

```diff
@@ -313,9 +313,9 @@ To start OSPF process you have to specify the OSPF router.
 Areas
 -----
 
-.. clicmd:: area A.B.C.D range A.B.C.D/M
+.. clicmd:: area A.B.C.D range A.B.C.D/M [advertise]
```

Comment:
With advertise/non-advertise option, cost can be configured too. Can you please check and update.

Reply:
Done in last push. Add ‘advertise cost’, ‘not-advertise’ is already there to remove all including ‘cost’.